static/app/utils/getPendingInvite.tsx

```diff
@@ -7,12 +7,22 @@ type PendingInvite = {
   url: string;
 };
 
+function isPendingInvite(invite: any): invite is PendingInvite {
```

Comment:
@evanpurkhiser small warning - this is a more strict check than before, but I think it's more accurate now. I had to update the tests to reflect that, but basically before, as long as a cookie "pending-invite" was set with any truthy value, we would assume it's the invite cookie. The change now is that we also check for the shape of the decoded cookie. I was tempted to throw an error if the isPendingInvite check failed, but this is usually annoying for customers to deal with bc they'd have to wipe their cookies. Maybe we could log a sentry exception here wdyt?